content/docs/sidebar.json

```
@@ -96,6 +96,10 @@
         "label": "DVC Files and Directories",
         "slug": "dvc-files-and-directories"
       },
+      {
+        "label": "Merge Conflicts",
+        "slug": "merge-conflicts"
+      },
```

```suggestion
      "merge-conflicts",
```
That should be enough ^ although maybe "merging-conflicts" would sound more actionable.

> p.s. we would probably move it into the How To subsection when it's introduced.